app/main.js

```diff
@@ -1,56 +1,55 @@
-import { Component } from 'react';
 import { Navigation } from 'react-native-navigation';
-import { AsyncStorage } from 'react-native';
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
 import thunk from 'redux-thunk';
 import { Provider } from 'react-redux';
-import { persistStore, autoRehydrate } from 'redux-persist';
-import { composeWithDevTools } from 'redux-devtools-extension';
 
 import Theme from 'config/theme';
 import { registerScreens } from 'screens';
 
 import * as reducers from 'redux-modules';
+import offline from './offline';
```

Comment:
Could you add offline as alias here https://github.com/Vizzuality/forest-watcher/blob/feature/dropdown-for-viirs-time/package.json#L73 ? 
Thanks !